lib/policies/jwt/jwt.js

```diff
@@ -21,7 +21,8 @@ module.exports = function (params) {
     }
 
     if (!params.checkCredentialExistence) {
-      return done(null, { id: 'anonymous' });
+      jwtPayload.consumerId && (jwtPayload.id = jwtPayload.consumerId);
```

Comment:
I think this is wrong.

The use case here is that, if we want to bypass the express-gateway identity system, we should just forward the payload and not make any assumption on it. Who's telling you that `jwtPayload.id` is always valorized and that its value corresponds to the consumer Id? That depends on your token provider.

Also, you're modifying the payload, while I think we should not touch it at all.

Reply:
Understand what you mean.
I think you are right.
thank you.